azure-pipelines.yml

```diff
@@ -33,26 +33,14 @@ stages:
           - script: |
               docker-compose run tests ./vendor/bin/phpunit --testsuite base-tests --debug
             env:
-              AWS_ECR_ACCESS_KEY_ID: $(AWS_ECR_ACCESS_KEY_ID)
-              AWS_ECR_REGISTRY_ACCOUNT_ID: $(AWS_ECR_REGISTRY_ACCOUNT_ID)
-              AWS_ECR_REGISTRY_REGION: $(AWS_ECR_REGISTRY_REGION)
-              AWS_ECR_REGISTRY_URI: $(AWS_ECR_REGISTRY_URI)
```

Comment:
Zůstaly jen secrety. Ostatní jsem vyhodil, protože to se propisuje automaticky a ani to tu nejde přetížit.
Pokud je nějakou normální variable potřeba přetížit, tak je to jako env předaný v `docker-compose run`